lua/weapons/weapon_gdo.lua

```diff
@@ -134,7 +136,43 @@ local function SendCode(EntTable)
 				end
 			end)
 		end
+		
+	else
+	
+		local self = EntTable;
+		
+		self.Stand = true;
+		self.gate:TriggerInput("Transmit","")
+		
+		local id = self:EntIndex()
+		
+		timer.Create("GDOTimer2"..id,0.5,0,function()
+			if (not IsValid(self.gate) or not IsValid(self.gate.Target)) then return end
```

Comment:
same - add IsValid(self) before self.gate. you need that check for that as for example if drop gdo and then remove it - error will appear.